### PR TITLE
Dcos 52817 use enum wherever possible in all fw configs

### PR DESCRIPTION
**Pull request by RezviyBelorus**
Cassandr, Elastic and HDFS config refactorings

### CODE REVIEW

File: frameworks/hdfs/universe/config.json [hunk 1]
```diff
@@ -1829,6 +1829,10 @@
         "hadoop_util_hash_type": {
           "type": "string",
           "description": "The default implementation of Hash. Currently this can take one of the two values: 'murmur' to select MurmurHash and 'jenkins' to select JenkinsHash.",
+          "enum": [
```
Comment (samvantran) [inline frameworks/hdfs/universe/config.json]: Please also add enums for 
- `disk_type`
- [logging_level](https://github.com/mesosphere/dcos-commons/blob/45625cee242f42988833d987087babe5cdd23481/frameworks/hdfs/universe/config.json#L299)
- [client_block_write_replace-datanode-on-failure_policy](https://github.com/mesosphere/dcos-commons/blob/45625cee242f42988833d987087babe5cdd23481/frameworks/hdfs/universe/config.json#L1200)

File: build_output.txt [hunk 2]
```diff
@@ -0,0 +1,648 @@
+Using provided AWS_PROFILE: 273854932432_Mesosphere-PowerUser
```
[the remark this answers is not in the document]
Comment (RezviyBelorus) [reply build_output.txt]: resolved

File: frameworks/cassandra/universe/config.json [hunk 3]
```diff
@@ -417,6 +417,11 @@
         "partitioner": {
           "type": "string",
           "description": "The partitioner used to distribute rows across the cluster. Murmur3Partitioner is the recommended setting. RandomPartitioner and ByteOrderedPartitioner are supported for legacy applications.",
+          "enum": [
+            "org.apache.cassandra.dht.Murmur3Partitioner",
+            "org.apache.cassandra.dht.RandomPartitioner",
+            "org.apache.cassandra.dht.ByteOrderedPartitioner"
+          ],
```
[the remark this answers is not in the document]
Comment (RezviyBelorus) [reply frameworks/cassandra/universe/config.json]: done

File: frameworks/hdfs/universe/config.json (hunk 1 above)
Comment (RezviyBelorus) [reply frameworks/hdfs/universe/config.json]: done

File: frameworks/cassandra/universe/config.json [hunk 4]
```diff
@@ -752,6 +766,10 @@
         "disk_optimization_strategy": {
           "type": "string",
           "description": "The strategy for optimizing disk reads",
+          "enum": [
+          "ssd",
+          "spinning"
```
[the remark this answers is not in the document]
Comment (RezviyBelorus) [reply frameworks/cassandra/universe/config.json]: done

**Pull request merged**